ChangeLog

```diff
@@ -59,13 +23,18 @@
   + tools: Allow more fine grained filtering of fence-history in crm_mon.
   + tools: Allow crm_rule to check some date_specs.
 
+- Fixes for regressions introduced in Pacemaker-2.0.0
+  + controller: properly detect remote node info requests
+
 - Changes since Pacemaker-2.0.3
   + build: improve checking headers for C++ fitness
   + build: restore build on latest OpenSUSE
   + libstonithd: Report an error getting history in output instead of
                  empty history
-  + libstonithd: Improve parameter handling for RHCS-style fence-agents
+  + libfencing, fencer: Improve parameter handling for RHCS-style fence-agents
                  - make parameter `action` shown as not required
+		 - add `port` or `plug` parameter according to metadata
+		 - `plug` parameter shown as non-required in the metadata
```

Comment:
looks like whitespace doesn't line up

FYI it's fine as is but also ok to just use "fencing:" to cover any changes related to fencing, since users don't really care what part of the code changed